docs/nomenclature.md

```diff
@@ -18,25 +27,81 @@ Mutant currently supports the following subjects:
 * Instance methods
 * Singleton (class) methods
 
-Other subjects (constants, class bodies for DSLs, ...) are possible but aren't
-implemented in the OSS version.
+Other subjects are possible (even project-specific subjects) but aren't implemented in the OSS version. Please get in touch with the authors. Some examples are:
+* Constants
+* Class bodies for DSLs
+
+The more subjects that mutant can alter in your project, the more mutations it can create, and so the higher your confidence you can have that your tests cover the semantics of your application. Please get in touch if you require subjects beyond those implemented in Mutant already - support may be available in the commercial version.
```

Comment:
This advises you to get in touch twice--feels slightly redundant.

Reply:
Yeah, I had meant to supplant the first one with the latter 😅